lib/Gitlab/Api/Issues.php

```diff
@@ -58,6 +58,16 @@ public function update($project_id, $issue_id, array $params)
      * @param int $issue_id
      * @return mixed
      */
+    public function remove($project_id, $issue_id)
```

Comment:
Mmmh... I'm confused: the gitlab API just tells to do a http delete on an issue id, which is like any other  object in gitlab.